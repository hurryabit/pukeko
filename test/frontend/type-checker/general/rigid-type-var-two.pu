f :: a -> b
f x = x
