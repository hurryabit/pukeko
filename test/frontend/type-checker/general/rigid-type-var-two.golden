test/frontend/type-checker/general/rigid-type-var-two.pu:2:1: mismatching types b and a
